### ISSUE TITLE
[SCM-765] jgit provider commits changes with system user instead of passed username

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-scm#972]
**[Bartholdi Dominik](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=imod)** opened **[SCM-765](https://issues.apache.org/jira/browse/SCM-765?redirect=false)** and commented

When using the jgit provider over http, it allows to define username / password like this:

```
mvn release:prepare release:perform -Dresume=false -Dusername=myuser -Dpassword=XXX
```

But the commit will not be done with the given user but with the system user who started the process, instead the passed user is only used for transport. Well that might be fine in the first thought, but it will mean in the git repository it is not recorded who really did make the release.

---

**Affects:** 1.9

**Comment by jira-importer** [comment]
**[Bartholdi Dominik](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=imod)** commented

PR placed https://github.com/apache/maven-scm/pull/16

**Comment by jira-importer** [comment]
**[Karl Heinz Marbaise](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=khmarbaise)** commented

seemed to be fixed by git commit: [037ba69bced119de1618526e6977940908eeb7bc](https://github.com/apache/maven-scm/commit/037ba69bced119de1618526e6977940908eeb7bc)

**Comment by jira-importer** [comment]
**[Bartholdi Dominik](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=imod)** commented

yeah, I finally got my commit credentials back working - thanks for checking :)
